server/schedule.py

```diff
@@ -112,9 +125,9 @@ def image(self, user, width, height):
             time = self._local_time.now(user)
         except DataError as e:
             raise ContentError(e)
-        today = time.replace(hour=0, minute=0, second=0, microsecond=0)
+
         while True:
```

Comment:
I don't think you need this loop anymore: `get_prev` should look back more than 1 day, so I don't think you need to manually keep decrease `time`.

Reply:
Yup all that makes sense. Thanks!